modules/installation-initializing.adoc

```diff
@@ -2,24 +2,22 @@
 //
 // * installing/installing_aws/installing-aws-customizations.adoc
 // * installing/installing_aws/installing-aws-network-customizations.adoc
-// * installing/installing_aws/installing-aws-private.adoc
 // * installing/installing_aws/installing-aws-vpc.adoc
 // * installing/installing_azure/installing-azure-customizations.adoc
 // * installing/installing_azure/installing-azure-network-customizations
-// * installing/installing_azure/installing-azure-private.adoc
 // * installing/installing_azure/installing-azure-vnet.adoc
 // * installing/installing_azure/installing-azure-user-infra.adoc
 // * installing/installing_gcp/installing-gcp-customizations.adoc
-// * installing/installing_gcp/installing-gcp-private.adoc
 // * installing/installing_gcp/installing-gcp-network-customizations.adoc
 // * installing/installing_gcp/installing-gcp-vpc.adoc
 // * installing/installing_gcp/installing-gcp-user-infra.adoc
 // * installing/installing_gcp/installing-restricted-networks-gcp.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
-// * installing/installing_openstack/installing-openstack-installer-user.adoc
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
 // * installing/installing_rhv/installing-rhv-customizations.adoc
-// * installing/installing_rhv/installing-rhv-default.adoc
```

Comment:
Interesting we had so many files incorrectly marked here. For example, I don't see this PR removing this module from the rhv-default.adoc file but it was removed in https://github.com/openshift/openshift-docs/pull/21910/files. I believe you are fixing that here (and amongst the other ones). Which makes this list suspect everywhere, and I am guessing you may have used your script to identify and fix?

Reply:
Yes, I created the script because of this module file and others like it in the Installation book. Where there are large lists of assemblies in module headings it is useful to be able to quickly check if the listings are still accurate.